osgp/shared/shared/src/main/java/org/opensmartgridplatform/shared/infra/jms/MessageMetadata.java

```diff
@@ -173,6 +181,8 @@ public String toString() {
                 ", domain=" + this.domain +
                 ", domainVersion=" + this.domainVersion +
                 ", ipAddress=" + this.ipAddress +
+                ", cellId=" + this.cellId +
+                ", btsId=" + this.btsId +
```

Comment:
Did not pay attention to this the first review, but I noticed it now.
The infra JMS MessageMetadata was set up for metadata with requests to be passed as JMS message properties.
These were values important to processing requests that were not part of the content of the request types.
Scheduling and retry properties, for instance are passes through SOAP headers, and the IP address is used because in some cases core can decide to put in the IP address of the device with the request or its gateway device.

As far as I understand the Cell ID and BTS ID are pieces of information that are passed in as part of the `SetCommunicationNetworkInformationRequest`. The IP address is in the request as well, but for setting the communication network information this is as part of the data, and not as part of the information passed by core to the protocol adapter through JMS metadata as described earlier in this comment.

Reply:
yes, this was added because it will be needed by the protocol adapter(s) to throttle/limit traffic to "stations/cells".

I assume this was discussed during the inter team refinements? 

So this PR just pushes the data to GXF, so it can later be sent as metadata to the protocol adapter where the "traffic manager" can decide whether to allow/queue the request based on the bts/cell.